README.rst

```diff
@@ -1,76 +1,51 @@
-|Build Status| |Codecov| |license|
+|Build Status| |Codecov| |license| |semantic-release| ## As many of these as you'd like to include, or others if you wish!
 
 frontend-app-communications
-=================================
+==============================
 
-Please tag **@edx/fedx-team** on any PRs or issues.  Thanks.
+Please tag **@edx/aperture-eng** on any PRs or issues.  Thanks!
```

Comment:
Is this supposed to be our GitHub team name? I think our team name is `edx-aperture`

Reply:
honestly I have no clue and took a guess!